apps/mobile/src/app/(tabs)/explore.tsx

```
@@ -72,53 +69,37 @@ const ExploreScreen = () => {
-                    icon="filter-variant"
-                    style={styles.filterButton}
-                >
-                    Filters
-                </Button>
-                {appliedFilters.length > 0 && (
-                    <Chip onClose={() => handleFilterApply([], null)} style={styles.filterChip}>
-                        {appliedFilters.length} filter{appliedFilters.length > 1 ? 's' : ''} applied
-                    </Chip>
-                )}
-            </View>
+            <View style={styles.filterContainer}></View>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)